content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,299 @@
+click the small black X at the end of the payslip line. If a specific :guilabel:`Salary Structure`
+or :guilabel:`Department` needs to be specified for the batch, select them form the drop downs.
+Click the :guilabel:`Generate` smart button to create the payslips for the batch.
+
+.. image:: payslips/generate-payslips.png
+   :align: center
+   :alt: Generata payslips for the new batch.
+
+.. note::
+   Both Selecting a department and/or salary structure will only display payslips that apply to
+   those selected parameters.
+
+If there are an errors or issues preventing the payslips from being generated, a detailed error will
+appear in the top right section. The box will disappear after several seconds, or click the X to
+close the alert. Make any modifications, such as removing any payslip lines that cannot be
+processed, then click the :guilabel:`Generate` smart button again.
+
+Once the payslips have been generated, the screen returns to the batch. Click the
+:guilabel:`Generate Draft Entry` smart button to chnge the payslips from draft mode to done. Once
+payslips have been generated, click the :guilabel:`Make Payment` smart button to process the
+payments. A pop up will appear so the banking information can be entered. Select the *Bank Journal*
+from the drop down, and enter the file name. Click the :guilabel:`Confirm` smart button when done,
+or click :guilabel:`Cancel`` to discard.
+
+.. image:: payslips/confirm-payslips.png
+   :align: center
+   :alt: Confirm the bank details for the new payslips.
+
+Generate comission payslips
+---------------------------
+
+Commission payslips can be generated in the batches section. Click on the batch or batches to create
+commissions payslips for. Click the :guilabel:`Generate Commission Payslips` smart button. A pop up
+appears to fill out.
+
+.. image:: payslips/commission.png
+   :align: center
+   :alt: Click the Generate Commission Payslips to create commission payslips.
+
+Click on the drop downs next to :guilabel:`Period` and select the period the payslips are being
+generated for. Using the left and right arrow, navigate to the correct month, and click on the date
+to select it.
+
+Select the department from the drop down menu. When the department is selected, the employees listed
+for that department will appear in the :guilabel:`Employee` section.
+
+Under the :guilabel:`Employee` section, enter the *Commission Amount* for each employee. To remove
+an employee, click the trash can icon to remove the line. Add a new entry by clicking on
+:guilabel:`Add a Line` and entering the employee name and the commission amount.
+
+Click the :guilabel:`Upload Your File` smart button to add a file. Any file type is accepted.
+
+Using the drop down, select either :guilabel:`Classic Commission` or  :guilabel:`Warrant` for the
+*Commission Type*. Classi is the most typical type of commission. Warrant is used for Belgian
+companies only.
```

Comment:
```suggestion
Using the drop-down menu beside the :guilabel;`Commission Type` field, select either :guilabel:`Classic Commission` or  :guilabel:`Warrant`. 

:guilabel:`Classic` is the most typical type of commission, while :guilabel:`Warrant` is primarily used for Belgian
companies only.
```

Personally, I tend to list the fields of a form/pop-up like this sequentially (top-to-bottom) - not a requirement, just a suggestion, so the reader doesn't have to have their eyes "jump around" too much, while reading the doc. Just a thought...